### PR TITLE
remove rustdoc warnings

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/librustc_mir_build/hair/pattern/_match.rs [hunk 1]
```diff
@@ -4,8 +4,8 @@
 //! This file includes the logic for exhaustiveness and usefulness checking for
 //! pattern-matching. Specifically, given a list of patterns for a type, we can
 //! tell whether:
-//! (a) the patterns cover every possible constructor for the type [exhaustiveness]
-//! (b) each pattern is necessary [usefulness]
+//! (a) the patterns cover every possible constructor for the type exhaustiveness
+//! (b) each pattern is necessary usefulness
```
Comment (ecstatic-morse) [inline src/librustc_mir_build/hair/pattern/_match.rs:8]: Use parens here.

**Comment by ecstatic-morse** [comment]
r=me with nit fixed.

@bors delegate+

File: src/librustc_mir_build/hair/pattern/_match.rs (hunk 1 above)
Comment (tshepang) [reply src/librustc_mir_build/hair/pattern/_match.rs:8]: What do those signify?

Comment (ehuss) [reply src/librustc_mir_build/hair/pattern/_match.rs:8]: I would say in this circumstance, the parenthesized word signifies how the word is being used in this context. That is, the phrase preceding the parenthetical is the definition for the word.

Comment (tshepang) [reply src/librustc_mir_build/hair/pattern/_match.rs:8]: I think I get it now... I thought those words were just continuations of the sentence

**Comment by tshepang** [comment]
@ecstatic-morse did I do the approval as you meant?

**Pull request merged**